### DISCUSSION TITLE
Current Resistor for LD24AJTA LED Driver

### DISCUSSION THREAD

**Discussion by ic-racer** [srozum/film_camera_tester#24] [Q&A]
Pulling this discussion out of my Build Thread.

The LD24AJTA LED driver board has a trimmer and a limiting fixed resistor.

As the board comes it has a 0.11 ohm resistor with current limited to 0.9A per PT4115 Datasheet. 

 Per the BOM and build instructions the limiting resistor (Rcs; circled in orange) should be changed to 0.91 ohm. Yielding a maximum current of 0.11A. 

Is this correct? Should it be 0.091 ohm?

0.091 ohm resistor should give 1.1A, yes?

Thank you in advance for you continued support of the project. 

![LED Driver ld2rajta](https://user-images.githubusercontent.com/128929440/228966771-b77d7b29-3102-4184-a340-85a948cd8684.jpg)
<img width="476" alt="Screen Shot 2023-03-30 at 4 30 34 PM" src="https://user-images.githubusercontent.com/128929440/228972581-9fd9fd12-9546-485e-9f69-9b5e5b5c466a.png">

<img width="241" alt="Screen Shot 2023-03-30 at 5 29 14 PM" src="https://user-images.githubusercontent.com/128929440/228969028-0d0a5bbb-b033-4d2f-b4fd-51c34f908771.png">

**Comment by srozum** [comment]
You are right, I made a typo. Rcs should be 0.091 Ohm, not 0.91 Ohm. I'll update Wiki.

However, not sure about your calculations of 1.1A. With 47µH inductor this board cannot provide more than 1A. Inductor saturates and both get very hot. With standard Rcs it outputs 0.7A, with 0.091 Ohm Rcs it outputs 0.9A.

**Reply by Pitichinaccio** [reply]
This was it that also I didn't understand:
If    Iout = 0,1 / Rcs (according to the datasheet)
then the current would be (theoretical) 1,1A  
-->  0,1 / 0,091 = 1,0989

Shouldn't it be 0,11 Ohm for getting Iout = 0,9A ?

**Comment by srozum** [comment]
![Screenshot 2023-03-30 at 4 30 36 PM](https://user-images.githubusercontent.com/56254/228987128-7c67c234-c98f-45ef-afbe-73bd78bf8c93.png)

I am not sure if you want to trust pictures from chinese sellers. Look at datasheet, and VA curve for setup with 47µH inductor. 0.7ish Amp at 12V with default Rcs 0.11 Ohm is what this board outputs, and it confirmed by measurements.

Rcs 0.091 Ohm is a safe compromise I found to keep driver cool and to produce required LED brightness without running it hot as well.

**Reply by ic-racer** [reply]
Thanks for the quick reply. I'll take your word on the output value. I just wanted to confirm if there was a typo on the resistor or not. 

Do you close the thread or do I?

**Comment by polygonfuture** [comment]
@srozum @ic-racer 

Can you confirm, this is a 1206 SMD resistor package correct?  I missed the small note in the BOM so now I need to order the resistor.

Also, I think it would be good to add explicit information on the resistor needing to be swapped / solder reworking to the assembly page along with the resistor package size, etc. 

The only mention is that small RCS line in the BOM which I totally missed until I found this thread.  If I hadn't found this thread I dont think I would have connected the dots about RCS on the board either.

**Comment by srozum** [comment]
Yes, it 1206, you can compare to the size of Rcs you have to replace.

Here is the part https://www.lcsc.com/product-detail/_UNI-ROYAL-Uniroyal-Elec-_C247523.html
